### ISSUE TITLE
Generate search index in multipage installations

### ISSUE THREAD

**Issue by Shoekrates** [contao/core#8881]
If you have multiple websites in one contao installation you can only generate the complete search index for all pages but not for one particular pagetree alone.
Could be a handy feature to set the starting page before generating the search index.

**Comment by rburch** [comment]
I agree. It would especially be useful if you have a really big site and then go live with a smaller one. You won't have to go back and re-index 1000s of pages just to index a 10 page site. I actually hit this issue on a multi-site for one of my clients.
